.github/workflows/multiflow.yml

```
@@ -1,62 +1,86 @@
+      uses: actions/checkout@v4
+    
+    - name: 🚚 Préparation du code
       id: composer-cache
       uses: actions/cache@v4
       with:
         path: vendor
         key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
         restore-keys: |
           ${{ runner.os }}-php-
-    - name: Install dependencies
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)